target/linux/ramips/image/mt7621.mk

```diff
@@ -1103,6 +1103,19 @@ define Device/ubnt_unifi-nanohd
 endef
 TARGET_DEVICES += ubnt_unifi-nanohd
 
+define Device/unielec_u7621-01-16m
+  $(Device/dsa-migration)
+  $(Device/uimage-lzma-loader)
+  IMAGE_SIZE := 16064k
+  DEVICE_VENDOR := UniElec
+  DEVICE_MODEL := U7621-01
+  DEVICE_VARIANT := 16M
+  DEVICE_PACKAGES := kmod-mt7603 kmod-mt76x2 kmod-usb-ledtrig-usbport \
+	kmod-usb3 -wpad-basic-wolfssl
+  SUPPORTED_DEVICES += u7621-01-256M-16M unielec,u7621-01-256m-16m
```

Comment:
This line can be dropped.

Reply:
The DEVICE_PACKAGES is that still not required? or is that set somewhere as default for this mt7621?

Reply:
SUPPORTED_DEVICES ...

Why am I never coming across with that one?